UKHO.PeriodicOutputService/UKHO.BESS.ConfigurationService.UnitTests/Services/ConfigurationServiceTests.cs

```diff
@@ -176,5 +187,200 @@ public void WhenContainerHasNoConfigs_ThenConfigIsNotAddedToList()
             dictionary.Add("Empty.json", InvalidEmptyJson);
             return dictionary;
         }
+
+        [Test]
+        public void WhenCheckConfigFrequencyAndSaveQueueDetailsSuccessful_ThenReturnsTrue()
```

Comment:
```suggestion
        public void WhenCheckConfigFrequencyAndSaveQueueDetailsIsSuccessful_ThenReturnsTrue()
```

Similarly for other unit test case as well

Reply:
Resolved